README.md

```diff
@@ -4,6 +4,37 @@ Overview
 This repository contain documentation and scripts that aim to help PC Engines
 apuX platform users and developers to customize firmware to their needs.
 
+Versioning scheme change
+------------------------
+
+Since version `v4.8.0.1` we changed versioning scheme skipping coreboot `v4.7`
```

Comment:
It's counter-intuitive `v4.8.0.1` is forked from `v4.8.1`

Reply:
@kmalkki can you explain how you get to that conclusion? I forked `rel_4.8.0.1` branch from `4.8` mainline tag which is on `ebdeb4d07da3`:

```
* d899716db7b4 - (HEAD -> rel_4.8.0.1, pcengines/rel_4.8.0.1) Merge pull request #165 from pcengines/configs_update (4 days ago) <Michał Żygowski>
*   63b64feae2a2 - (pcengines/develop) Merge branch 'master' into develop (7 days ago) <Piotr Król>
|\
| * ebdeb4d07da3 - (tag: 4.8) Documentation/Intel/NativeRaminit: Style fixes (3 weeks ago) <Patrick Rudolph>
| * fd470f716370 - (tag: v4.7.0, tag: coreboot-4.7.x, tag: 4.7) vx900: fix format strings for DEBUG_RAM_SETUP=y (5 months ago) <Lubomir Rintel>
* | c3314b1c56fd - (tag: v4.6.9, pcengines/release, pcengines/coreboot-4.6.x) Merge pull request #143 from pcengines/rel_v4.6.9 (4 weeks ago) <Michał Żygowski>
```

`v4.8.1` branch from above commit by adding `5f0b80b880b6` and `6794ce02d452`.